doc/source/examples/examples.rst

```diff
@@ -0,0 +1,21 @@
+Sphinx design
+=============
+The rendering of sphinx design with ansys sphinx theme. To use this see 
+the full document of `sphinx design <https://sphinx-design.readthedocs.io/en/latest/index.html>`_.
+
+.. jinja:: examples
+
+    {% for filename in inputs_examples %}
+    {% set title = filename.split('.')[0] %}
+
+    {{ title[0].upper() }}{{ title[1:] }}
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+        .. literalinclude:: sphinx_examples/{{ filename }}
+           :language: bash
```

Comment:
Language should be `rst`:

```suggestion
           :language: rst
```

I know that the original examples are in `txt` but their content is `rst`. If we need to modify the file extension when downloading those, let's do that unless it causes problems when building the docs.